Dockerfile

```
@@ -3,7 +3,7 @@
 FROM condaforge/mambaforge:latest AS builder
 
 RUN mamba update --yes --all
-RUN mamba install --yes "boa" "conda-verify"
+RUN mamba install -c conda-forge --yes "boa"
```

```suggestion
RUN mamba install --yes "boa"
```
Is the channel specification necessary here?

---

i think not